go.mod

```diff
@@ -2,4 +2,8 @@ module github.com/m-lab/ndt7-client-go
 
 go 1.12
 
-require github.com/gorilla/websocket v1.4.0
+require (
+	github.com/gorilla/websocket v1.4.0
+	github.com/m-lab/ndt-server v0.13.2-0.20190920160430-5b7940b72f0d
```

Comment:
What's this extra stuff here? I'd like to be depend on a real tagged version of ndt-server...